quickstarts/microsoft.azurestackhci/create-cluster/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "creates an Azure Stack HCI 23H2 cluster",
+  "description": "This template creates an Azure Stack HCI 23H2 cluster using an ARM template.",
+  "summary": "This template creates an Azure Stack HCI 23H2 cluster using an ARM template.",
+  "githubUsername": "troettinger",
+  "validationType": "Manual",
```

Comment:
Why is manual validation required?

Reply:
Validation can be automated for sure but requires hardware wired correctly as the template does deploy a Windows Failover cluster using hardware. Hence i did pick manual validation as its not deploying a bunch of VMs.